tools/data/resize_video.jl

```diff
@@ -0,0 +1,72 @@
+using DelimitedFiles
```

Comment:
I suggest rewriting this script with python.

Reply:
yeah agree. The above script meant to be a reference only.

Reply:
Will do.